### ISSUE TITLE
Making pictures a movie. (Shall we improve the last hackday project?)

### ISSUE THREAD

**Issue by linan7788626** [linan7788626/hackday_10302015#2]
Dear @rbiswas4 and @danielsf,

It has been a long time since (last October) we played with 
the interesting project for the hackday at Argonne. 
[Einstein Amazeball](https://github.com/DarkEnergyScienceCollaboration/Twinkles/issues/50)
Are you still interested in improving it a little bit? I think it would be helpful for the twinkles simulation. 

Originally, the basic idea is that : 
1. Rahul produces a supernova light curve (with mjd and magnitude)
2. Scott will produce the input files for PhoSim using OpSim according to the mjd from Rahul.
3. I will produce lensed supernova and lensed galaxies.
4. Input corresponding lensed images and Cadences into PhoSim, 
   we produce a set of mock observations.
5. At last, compact these pictures to a movie. 

I remember that we just made one image only, because of the low efficiency of running PhoSim on my laptop. Recently, I am trying to produce all the images, but it seems that I forgot some details of the light curve from @rbiswas4  and the cadences from @danielsf. Do you guys mind if I ask you questions about this? Truth me, it will not take much time. 

If you both would like to help me, I will ask questions under this issue. And @danielsf , do you mind if I add you as a collaborator of this project? 

Should you have any questions, suggestions, or comments, please feel free to let me know.
Thank you so much!

All the best,
Nan Li

**Comment by rbiswas4** [comment]
Hi Nan,

Sure ... I don't recall a lot of the details of what we had agreed on, but
I will try.

Thanks,
Rahul

**Comment by linan7788626** [comment]
Hi @rbiswas4 ,

I read you ipython notebook again, I think the light curve you give me is fair enough for this project. but one thing I want to double check is that : all data of the light curve is observed value, right?

Hi @danielsf ,  do you happen to know how to install lsst_stack and lsst_sims module on NERSC, for example Edison or Cori?

**Comment by rbiswas4** [comment]
On Wed, Jan 27, 2016 at 12:41 PM, Nan Li notifications@github.com wrote:

> Hi @rbiswas4 https://github.com/rbiswas4 ,
> 
> I read you ipython notebook again, I think the light curve you give me is
> fair enough for this project. but one thing I want to double check is that
> : all data of the light curve is observed value, right?

Can you point me to the notebook?

**Comment by linan7788626** [comment]
Here you go:

https://github.com/linan7788626/hackday_10302015/blob/master/SN_for_hack.ipynb

**Comment by rbiswas4** [comment]
Hi Nan,

Ok, those are 'observed' in the sense of being the expected values when
observed from the earth. Obviously there is Poisson scatter (sky, source
photons) which will make the light curve less smooth, but that has not been
included. Usually that is taken into account when creating the image, so
adding it here will result in double counting. But if in this particular
project that is necessary, we can do that too, but will need additional
information.

Thanks,
Rahul

**Comment by linan7788626** [comment]
Hi @rbiswas4 ,

Thank you for your confirmation, we will add the Poisson scatter at the last step, which is inputting the lensed image into PhoSim. So far, you light curve is good enough.  

There is mjd in your light curve, so Scott can use these mjds to produce OpSim cadences for PhoSim for different data points of your light curve. Do I understand right? Or do we need anything more to produce OpSim cadences, besides the mjds of your light curve? 

Thanks,
Nan

**Comment by rbiswas4** [comment]
@linan7788626 

> There is mjd in your light curve, so Scott can use these mjds to produce OpSim cadences for PhoSim for different data points of your light curve. Do I understand right? 

Yes, he can. And the enigma OpSim database had measurements at those MJD in the field we were talking about. However, I think we are at the brink of a change to the projectwide OpSim run (it will be Kraken_1042) and that may not work at the same mjd values.

Or do we need anything more to produce OpSim cadences, besides the mjds of your light curve?

If we are doing different MJDs we will have to do this differently.

**Comment by linan7788626** [comment]
Hi @rbiswas4 
Got it, thanks, Anyway, on my side,  I will produce lensed supernovas and lensed galaxies for all data points of your light curve first. we can discuss the MJD and OpSim run when Scott is available. So far, 
I think all questions about your light curve are answered. Thanks a lot.

All the best,
Nan
